packages/headless/doc-parser/src/code-sample-resolver.ts

```diff
@@ -7,6 +7,7 @@ const packageJson = JSON.parse(readFileSync(packageJsonPath, 'utf-8'));
 export interface SamplePaths {
   react_class?: string[];
   react_fn?: string[];
+  react_usage?: string[];
```

Comment:
Instead of a new key, why not place the path inside the `react_fn` and `react_class` property? I think this will be easier for the doc team to display. If a reader chooses react-class or react-fn, they will see the same sample when looking at the context controller.